fluss-client/src/main/java/org/apache/fluss/client/admin/FlussAdmin.java

```diff
@@ -255,7 +255,7 @@ public CompletableFuture<Void> alterTable(
                 .setTablePath()
                 .setDatabaseName(tablePath.getDatabaseName())
                 .setTableName(tablePath.getTableName());
-        return gateway.alterTableProperties(request).thenApply(r -> null);
+        return gateway.alterTable(request).thenApply(r -> null);
```

Comment:
update java doc of method `AdminGateway#alterTable`
Currently, it's `the request to alter the configs of a table.`